pulsar-io/redis/src/main/java/org/apache/pulsar/io/redis/sink/RedisSink.java

```diff
@@ -49,7 +49,7 @@
     configClass = RedisSinkConfig.class
 )
 @Slf4j
-public class RedisSink<T> implements Sink<T> {
+public class RedisSink<T> implements Sink<byte[]> {
```

Comment:
Should it then be `RedisSink` instead of `RedisSink<T>` ?

Reply:
Done.